src/main/java/fr/novia/zaproxyplugin/ZAProxy.java

```diff
@@ -831,6 +863,31 @@ private void spiderURL(final String url, BuildListener listener, ClientApi zapCl
 			Thread.sleep(1000);
 		}
 	}
+
+	/**
+	 * Search for all links and pages on the URL and raised passives alerts
+	 *
+	 * @param url the url to investigate
+	 * @param listener the listener to display log during the job execution in jenkins
+	 * @param zapClientAPI the client API to use ZAP API methods
+	 * @throws ClientApiException
```

Comment:
Please avoid empty Javadoc entries. It will become a problem after the migration to JDK8